src/App.js

```diff
@@ -1,24 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
```

Comment:
- [ ] [OPTIONAL] React throw a Warning in the console, kindly consider fixing the issues so the app is Error/Warning free